app/lib/image_resizer.rb

```diff
@@ -0,0 +1,22 @@
+module ImageResizer
```

Comment:
I would imagine this to be in `app/services` :)
Something like `Images::Resize.call`

Reply:
Thanks for this suggestion! I'm going to do this on the next PR

Reply:
@maestromac We recently started adding more models that aren't backed by `ActiveRecord` to `app/models`. This often raises the question of where something should go. Something that served me well previously is following a  separation between actions (verbs) and things (nouns). So the question becomes does this object represent an action or a thing? In the first case, I'd put it in `app/services/` and call it `ResizeImage` (sometimes referred to as the function object pattern) in the second case I'd put it in `app/models` and keep the current name. In this specific instance, I'd probably go for a "verb" in services.

You don't have to follow this of course, but thinking about this way usually helps me clarify where I want to put something.

Reply:
Thanks you for that @citizen428 !

I've always been a bit confused between what to put in services vs models. There is a plan to implement an `Image` model so my plan is to eventually move it to `app/models` but you just gave me a new perspective. This is an action and should remain `services`.

Reply:
You're welcome @maestromac 😃 If you already know you are going to add an `Image` model, consider namespacing the actions in `app/services`, e.g. `Images::Resize`